### PR TITLE
chore(test/libsinsp_e2e,ci): port libsinsp_e2e tests to use python3.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI
/area tests

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

Extracted from #2064 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/libsinsp_e2e/resources/unix_client_server.py [hunk 1]
```diff
@@ -27,17 +26,15 @@
...
 
 else:
-  sys.stdout = os.fdopen(sys.stdout.fileno(), 'w', 0)
-
   if os.path.exists(NAME):
     client = socket.socket( socket.AF_UNIX, socket.SOCK_STREAM )
     client.connect(NAME)
 
-    print "STARTED"
+    print(STARTED)
```
Comment (Molter73) [inline test/libsinsp_e2e/resources/unix_client_server.py:33]: [nitpick] A variable for two print messages is a bit overkill, don't you think?
```suggestion
    print("STARTED")
```

**Review by Molter73 — APPROVED** [review]
/lgtm

File: test/libsinsp_e2e/resources/unix_client_server.py (hunk 1 above)
Comment (FedeDP) [reply test/libsinsp_e2e/resources/unix_client_server.py:33]: I don't think this is worth the amend :) actually i like it better like this; de gustibus eheh

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**